src/main/java/adris/altoclef/tasksystem/ITaskLimitsSlots.java

```diff
@@ -0,0 +1,28 @@
+package adris.altoclef.tasksystem;
+
+import adris.altoclef.AltoClef;
+
+/**
+ * Some tasks (mainly tasks that open containers) will break if they don't require
+ * that the 2x2 crafting grid is empty. Tasks which implement this interface
+ * require implementers to maintain an empty crafting grid.
+ *
+ * This interface let's a task declare that it MUST have certain slots clear before it can execute.
+ */
+public interface ITaskLimitsSlots {
```

Comment:
I don't see why we need an interface, this can probably be deleted.

Reply:
Renamed and re-did better